packages/components/src/components/accordion/accordion.tsx

```diff
@@ -81,18 +81,29 @@ export class Accordion {
   /** Emitted when accordion state is changed. */
   @Event({ bubbles: false }) public update: EventEmitter<AccordionUpdateEventDetail>;
 
+  @State() private internalOpen: boolean = false;
```

Comment:
Usually we would make the `open` prop mutable and directly change it instead of creating a separate internal state. That way the open prop also always reflects the current state to the outside. If used in a controlled manner the prop still can be changed with the update event.